proposals/devcontainer-features.md

```diff
@@ -91,35 +51,67 @@ Options
+Features are referenced in a user's [`devcontainer.json`](/docs/specs/devcontainer-reference.md#devcontainerjson) under the top level `features` object. 
 
-If the application finds a `devcontainer-collection.json` file in the root of a downloaded tar file, then it uses that file to find the particular feature that will be executed.
+A user can specify an arbitrary number of features.  At build time, these features will be installed in an order defined by a combination of the [installation order rules and implementation](#Installation-Order). 
 
-In addition to the list of features included, the file includes the following properties.
+A single feature is provided as a key/value pair, where the key is the feature identifier, and the value is an object containing "options" (or empty for "default").  Each key in the feature object must be unique.
 
-| Property | Type | Description |
-| :--- | :--- | :--- |
-| name | string | Name of the collection. |
-| reference | string | Reference information of the repository and path where the code is stored. |
-| version | string | Version of the code. |
+These options are sourced as environment variables at build-time, as specified in [Option Resolution](#Option-Resolution).
 
-In most cases, the `devcontainer-collection.json` file can be generated automatically at the moment of creating a release.
 
-## devcontainer.json properties
+Below is a valid `features` object provided as an example.
+```jsonc
+"features": {
+  "ghcr.io/user/repo/go:1": {},
+  "https://github.com/user/repo/releases/devcontainer-feature-go.tgz": { 
+        "optionA": "value" 
+  },
+  "./myGoFeature": { 
+        "optionA": true,
+        "optionB": "hello",
+        "version" : "1.0.0"
+  }
+}
+```
+
+An option's value can be provided as either a `string` or `boolean`, and should match what is expected by the feature in the `devcontainer-feature.json` file.
+
+As a shorthand, the value of a `feature` can be provided as a single string. This string is mapped to an option called `version`.  In the example below, both examples are equivalent. 
 
-Features are referenced in `devcontainer.json` , where the `features` tag consists of an object tag starting with the id of the feature and including the values of the options to pass to the feature itself.
+```jsonc
+"features": {
+  "ghcr.io/owner/repo/go": "1.18"
+}
+```
+```jsonc
+"features": {
+  "ghcr.io/owner/repo/go": {
+    "version": "1.18"
+  }
+}
+```
+
+### Referencing a feature 
 
+The `id` format specified dicates how a supporting tool will locate and download a given feature. `id` is one of the following:
+
+| Type | Description | Example |
+| :--- | :--- | :--- |
+| `<oci-registry>/<namespace>/<feature>[:<semantic-version>]` | Reference to feature in OCI registry(*) | ghcr.io/user/repo/go:1 |
```

Comment:
Same here, mention that namespece should be organization/repo? Or at least organization?

Reply:
Added note above.